src/doc/src/reference/registries.md

```diff
@@ -90,576 +93,9 @@ has a separate table for each registry, for example:
-            // The unique username of the owner.
-            "login": "github:rust-lang:core",
-            // Name of the owner.
-            // This is optional and may be null.
-            "name": "Core",
-        }
-    ]
-}
-```
-
-##### Owners: Add
-
-- Endpoint: `/api/v1/crates/{crate_name}/owners`
-- Method: PUT
-- Authorization: Included
-
-A PUT request will send a request to the registry to add a new owner to a
-crate. It is up to the registry how to handle the request. For example,
-[crates.io] sends an invite to the user that they must accept before being
-added.
-
-The request should include the following JSON object:
-
-```javascript
-{
-    // Array of `login` strings of owners to add.
-    "users": ["login_name"]
-}
-```
-
-A successful response includes the JSON object:
-
-```javascript
-{
-    // Indicates the add succeeded, always true.
-    "ok": true,
-    // A string to be displayed to the user.
-    "msg": "user ehuss has been invited to be an owner of crate cargo"
-}
-```
-
-##### Owners: Remove
-
-- Endpoint: `/api/v1/crates/{crate_name}/owners`
-- Method: DELETE
-- Authorization: Included
-
-A DELETE request will remove an owner from a crate. The request should include
-the following JSON object:
-
-```javascript
-{
-    // Array of `login` strings of owners to remove.
-    "users": ["login_name"]
-}
-```
-
-A successful response includes the JSON object:
-
-```javascript
-{
-    // Indicates the remove succeeded, always true.
-    "ok": true
-}
-```
-
-#### Search
-
-- Endpoint: `/api/v1/crates`
-- Method: GET
-- Query Parameters:
-    - `q`: The search query string.
-    - `per_page`: Number of results, default 10, max 100.
-
-The search request will perform a search for crates, using criteria defined on
-the server.
-
-A successful response includes the JSON object:
-
-```javascript
-{
-    // Array of results.
-    "crates": [
-        {
-            // Name of the crate.
-            "name": "rand",
-            // The highest version available.
-            "max_version": "0.6.1",
-            // Textual description of the crate.
-            "description": "Random number generators and other randomness functionality.\n",
-        }
-    ],
-    "meta": {
-        // Total number of results available on the server.
-        "total": 119
-    }
-}
-```
-
-#### Login
-
-- Endpoint: `/me`
-
-The "login" endpoint is not an actual API request. It exists solely for the
-[`cargo login`] command to display a URL to instruct a user to visit in a web
-browser to log in and retrieve an API token.
-
 [Source Replacement]: source-replacement.md
-[`cargo login`]: ../commands/cargo-login.md
-[`cargo package`]: ../commands/cargo-package.md
+[Running at Registry]: running-a-registry.md
 [`cargo publish`]: ../commands/cargo-publish.md
-[alphanumeric]: ../../std/primitive.char.html#method.is_alphanumeric
+[`cargo package`]: ../commands/cargo-package.md
+[`cargo login`]: ../commands/cargo-login.md
 [config]: config.md
```

Comment:
Can you add this to the bottom of the file so that links to the old sections don't get broken, and go to the right place?

```suggestion
[config]: config.md

<script>
(function() {
    var fragments = {
        "#running-a-registry": "running-a-registry.html",
        "#index-format": "registry-index.html",
        "#web-api": "registry-web-api.html",
        "#publish": "registry-web-api.html#publish",
        "#yank": "registry-web-api.html#yank",
        "#unyank": "registry-web-api.html#unyank",
        "#owners": "registry-web-api.html#owners",
        "#owners-list": "registry-web-api.html#owners-list",
        "#owners-add": "registry-web-api.html#owners-add",
        "#owners-remove": "registry-web-api.html#owners-remove",
        "#search": "registry-web-api.html#search",
        "#login": "registry-web-api.html#login",
    };
    var target = fragments[window.location.hash];
    if (target) {
        var url = window.location.toString();
        var base = url.substring(0, url.lastIndexOf('/'));
        window.location.replace(base + "/" + target);
    }
})();
</script>
```

Reply:
Great! I was a bit worried about breaking links.